apps/showcase/src/services/webcontainer/webcontainer-runner.ts

```diff
@@ -147,10 +174,35 @@ export class WebContainerRunner {
     const process = await instance.spawn(command, args, {cwd: cwd});
     this.commandOutput.process.next(process);
     const exitCode = await process.exit;
-    if (exitCode !== 0) {
+    if (exitCode === 0) {
```

Comment:
Shouldn't we prefer a switch case, in case we have to handle others exitCode or it should never happen ?

Reply:
I think it's a matter of preference, I don't see any good reason to choose one or the other.
I could argue that with `if` we can more easily include ranges `if (existCode >= 100 && exitCode < 200)`
you can also do it with `switch(true) { case (existCode >= 100 && exitCode < 200): ... }` but at the end it looks more like an `if` than a `switch/case`